docs/pages/app-signing/app-credentials.mdx

```diff
@@ -118,7 +118,7 @@ When you use the `eas credentials` command to delete your credentials, this only
 
 ### Re-signing new credentials
 
-You can use `eas build:resign` to codesign an existing **.ipa** for iOS to a new ad hoc provisioning profile. This helps reduce time when distributing internally &mdash; for example, if you want to add a new test device to an existing build, you can use this command to update the provisioning profile to include the device without rebuilding the entire app from scratch.
+You can use `eas build:resign` to resign an existing **.ipa** for iOS to a new ad hoc provisioning profile. This helps reduce time when distributing internally &mdash; for example, if you want to add a new test device to an existing build, you can use this command to update the provisioning profile to include the device without rebuilding the entire app from scratch.
```

Comment:
```suggestion
You can use `eas build:resign` to codesign an existing **.ipa** for iOS to a new ad hoc provisioning profile. This helps reduce time when distributing internally &mdash; for example, if you want to add a new test device to an existing build, you can use this command to update the provisioning profile to include the device without rebuilding the entire app from scratch.
```

@albert-schilling, the terminology Apple uses in their official documentation for this process is Code signing: https://developer.apple.com/documentation/security/code_signing_services. Sticking to that terminology to describe what the command does, I'm reverting this suggestion.